### PR TITLE
Improve chunk_gated_delta_rule_fwd test coverage

### PR DESCRIPTION
**Pull request by Schopenhauer-loves-Hegel**
Add missing accuracy test and performance benchmark for chunk_gated_delta_rule_fwd operator.

### CODE REVIEW

File: benchmark/test_FLA/test_chunk_gated_delta_rule_fwd.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import pytest
+import torch
+import torch.nn.functional as F
+
+import flag_gems
+from benchmark.base import Benchmark
+
+
+class ChunkGatedDeltaRuleFwdBenchmark(Benchmark):
+    DEFAULT_DTYPES = [torch.bfloat16]
```
Comment (tengqm) [inline benchmark/test_FLA/test_chunk_gated_delta_rule_fwd.py]: why only bf16 is tested?

File: benchmark/test_FLA/test_chunk_gated_delta_rule_fwd.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import pytest
```
Comment (tengqm) [inline benchmark/test_FLA/test_chunk_gated_delta_rule_fwd.py:1]: remove `_perf` from file name.

File: benchmark/test_FLA/test_chunk_gated_delta_rule_fwd.py (hunk 1 above)
Comment (lukatao) [reply benchmark/test_FLA/test_chunk_gated_delta_rule_fwd.py]: fixed

**Review by w1120029931-bit — APPROVED** [review]
